integration-cli/docker_utils.go

```diff
@@ -550,6 +550,10 @@ func pullImageIfNotExist(image string) (err error) {
 	return
 }
 
+func dockerCmdWithError(c *check.C, args ...string) (string, int, error) {
```

Comment:
I think the parameter "c" can be omitted. And we can call this function in "dockerCmd".